contracts/token/ERC721/extensions/ERC721Consecutive.sol

```diff
@@ -159,7 +159,7 @@ abstract contract ERC721Consecutive is IERC2309, ERC721 {
     }
 
     /**
-     * @dev Used to offset the first token id in {_nextConsecutiveId}
+     * @dev Used to offset the first token id in `_nextConsecutiveId`
```

Comment:
Note: the rational here is that `_nextConsecutiveId` is a private function, and therefore doesn't appear in the docs